tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27, py35
+
+[testenv]
+commands = py.test
+deps =
+    pytest
```

Comment:
Upgrade your toxicity by including pytest-cov in your dependencies, and using this command instead:
`py.test --cov={toxinidir} --cov-report term-missing`

I think that's right, anyway. Double check by reading the tox and coverage docs.